stor/dx.py

```diff
@@ -809,7 +822,7 @@ def read_object(self):
             bytes: the raw bytes from the object on DX.
         """
         if not self.resource:
-            raise ValueError('Can only read_object() on a file path, not a project')
+            raise ValueError('Can only call read_object() on a file path')
```

Comment:
might be nice to do something like `+ (', not a project' if self.project else '')` here

Reply:
Added